docs/REST API/rest-api.md

```diff
@@ -9082,8 +9082,45 @@ Retrieves a list of all CVEs affecting software and/or OS versions.
     "has_previous_results": false
   }
 }
+
+```
+##### Default response (no results, known vulnerability)
+
+The search query matches a vulnerability (CVE) that's in Fleet's vulnerability database but hasn't been found on any hosts yet. Learn more [here](../Using%20Fleet/Vulnerability-Processing.md).
+
+`Status: 200`
+
+```json
+{
+  "vulnerabilities": null,
+  "known_vulnerability": true,
+  "count": 0,
+  "counts_updated_at": "2024-02-02T16:40:37Z",
+  "meta": {
+    "has_next_results": false,
+    "has_previous_results": false
+  }
```

Comment:
https://nvd.nist.gov/vuln/detail/CVE-2024-2325

Should probably add something returning from the API if it searched an exact match or not. 

For example:
Fleet instance at company A might search WITHOUT quotes: CVE-2024-2325
Say they get no response, we'll they got no response for CVE-2024-2325 but also all similar CVEs like CVE-2024-2325x. So UI would show a generic no results found empty state.
Fleet instance at company B might search WITH quotes: "CVE-2024-2325"
Say they get no response, they only got no response for CVE-2024-2325. Here there would be a more specific no results found empty state that has whether or not the vulnerability is known.